Kernel/Arch/aarch64/Registers.h

```diff
@@ -487,4 +487,39 @@ struct ESR_EL1 {
 };
 static_assert(sizeof(ESR_EL1) == 8);
 
+// https://developer.arm.com/documentation/ddi0601/2020-12/AArch64-Registers/DAIF--Interrupt-Mask-Bits?lang=en
+// DAIF, Interrupt Mask Bits
+struct DAIF {
+    u64 : 6;
```

Comment:
Maybe add comments saying what these do (`// Reserved`, `// Mask FIQ interrupts`, etc), or maybe even give the fields more descriptive names. (There's value in using the names from the docs for greppability etc, but `F` is so short that it's hard to search for anyways.)

Reply:
I think as long as there is a link to the documentation, it is fine to use the short names that are also used in the documentation. The style in this file is a bit all over the place anyway, and it's probably good to refactor this some time (but that can be for another PR).